javaslang/generator/Generator.scala

```diff
@@ -1098,6 +1098,17 @@ def generateMainClasses(): Unit = {
               public T$j _$j() {
                   return _$j;
               }
+
+              /$javadoc
+               * Wither of the ${j.ordinal} element of this tuple.
```

Comment:
Please change it to `Sets the ${j.ordinal} element of this tuple to the given {@code value}.`